frontend/public/components/app-contents.tsx

```diff
@@ -413,8 +413,19 @@ const AppContents: React.FC<{}> = () => {
         exact
         kind="Route"
         loader={() =>
-          import('./routes/create-route' /* webpackChunkName: "create-route" */).then(
-            (m) => m.CreateRoute,
+          import('./routes/RoutePage' /* webpackChunkName: "create-route" */).then(
+            (m) => m.RoutePage,
+          )
+        }
+      />
+
+      <LazyRoute
+        path="/k8s/ns/:ns/routes/:name/edit"
+        exact
+        kind="Route"
+        loader={() =>
+          import('./routes/RoutePage' /* webpackChunkName: "create-route" */).then(
```

Comment:
Even though it is a same underlying page components, Its best to have different chunk names for create and edit pages.
```suggestion
          import('./routes/RoutePage' /* webpackChunkName: "edit-route" */).then(
```

Reply:
Updated.